lib/common/presentation/views/home_screen.dart

```diff
@@ -40,7 +41,8 @@ class HomeScreen extends HookWidget {
           final innerIsScrolled,
         ) =>
             const [WebsiteHeader()],
-        body: SingleChildScrollView(
+        body: const VisitUs(),
+        /*SingleChildScrollView(
```

Comment:
Why is this all still scrolled out? Is the VisitUs() widget now the homepage? 

set it back to normal

Reply:
This page should ideally be unedited